src/renderer/components/+preferences/app-preference-tab/app-preference-tab-registration.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+export interface AppPreferenceTabRegistration {
```

Comment:
Why this needs to have separate file?

Reply:
No specific reason to keep it in separate file.